src/types.ts

```diff
@@ -471,8 +471,10 @@ export interface RevolutCheckoutCardField extends RevolutCheckoutInstance {
 export interface PaymentRequestInstance {
   /** Render the payment request button */
   render: () => Promise<void>
-  /** Check if user can make payment via a supported payment request method  */
-  canMakePayment: () => Promise<boolean>
+  /** Check if user can make payment via a supported payment request method. 
+    * Returns a promise with the decided method or null if cannot make payment.
```

Comment:
Can we discard the extra description? 
For consistency, the tsdoc comments in this file just describe what the method does. 
If you wanna be a purist you'd introduce `@returns` to describe what's returned

I'd say we just leave as before: 

```js
// Check if user can make payment via a supported payment request method
```